pkgs/tools/misc/findutils/default.nix

```diff
@@ -10,7 +10,13 @@ stdenv.mkDerivation rec {
     sha256 = "178nn4dl7wbcw499czikirnkniwnx36argdnqgz4ik9i6zvwkm6y";
   };
 
-  patches = [ ./memory-leak.patch ./no-install-statedir.patch ];
+  patches = [
+    ./memory-leak.patch
+    ./no-install-statedir.patch
+
+    # prevent tests from failing on old kernels
```

Comment:
It might be also interesting to add here, what your kernel version was (also you don't know the exact one).

Reply:
Ancient.

Reply:
@Mic92 These are branches supported by RedHat, they come with backported features as well. I believe the "-754" version came out last month.